src/main/java/dk/cngroup/lentils/service/validators/MandatoryFieldsDependentOnOtherValidator.java

```diff
@@ -35,9 +35,9 @@ public boolean isValid(final Object object, final ConstraintValidatorContext con
             Object matchFieldValue2 = getFieldValue(object, matchField2);
             Object matchFieldValue3 = getFieldValue(object, matchField3);
             MultipartFile file = (MultipartFile) matchFieldValue1;
-            return baseFieldValue == ImageSource.FILE && !file.isEmpty()
-                    || baseFieldValue == ImageSource.FILE && !matchFieldValue3.equals("")
-                    || baseFieldValue == ImageSource.WEB && !matchFieldValue2.equals("")
+            return baseFieldValue == ImageSource.LOCAL && !file.isEmpty()
```

Comment:
Co je prosim matchFiledValue3? Neslo by to vsechno lepe pojmenovat? A ma se tady urcite porovnavat pres == a ne pres .equals()?

Reply:
prejmenovala jsem to. To equals jsem tam puvodne mela, ale nefungovalo mi to, tak jsem to zkousela takhle, a nevim proc to fungovalo. Ale ted to s equals funguje, tak to asi bylo v necem jinem - je to tam